### ISSUE TITLE
New promo image ideas

### ISSUE THREAD

**Issue by GaurangTandon** [GaurangTandon/ProKeys#54]
Based off inspirations from other text expanders.

The problem with the current promo images is that: they reveal too less details about our software. This makes the user watch the video and read the description - both of which the user is reluctant about.

The following details should be promoted in the next set of promo images:

### **Promo image 1**
Should highlight **"extreme --something--"**
1. **intelligent** rich and plain text support - swaps between pasting rich content/plain text according to element operating upon
2. replacement patterns for date, time, clipboard
3. context menu integration
4. built-in forms/placeholders
5. time shifting

with a background image of a sample snippet showing off everything above along with a transformed image, like this - https://lh3.googleusercontent.com/0GXkNPC9UHW-ZBra-NmnCkEzwf4nzvQI3uLjomQgh5PRxRe-wL0TeEFVkXPFZEy0aGmdCgrD=s640-h400-e365-rw (note the left part)

### **Promo image 2**
Should highlight **"extremely friendly software"**
1. no account/signup required. just download and start using
2. several pre-loaded snippets
3. user-friendly interface
4. full help page as well as a responsive support team.
5. optional testimonial: "I had to contact support because another extension was causing problems but I soon resolved it and The Developer was quick and polite with his responses!" (Richard Myers)

### **Promo image 3**
Should highlight **"extreme organization"**
1. nested folder support
2. sorting of folders and snippets by name and date
3. bulk actions on snippets
4. past 20 revisions backup 
5. import/export data
6. mac, windows, linux sync support

### **Promo image 4**
Should highlight **"extreme --something--"**
Make this promo image by combining the "works on all websites" promo image with the following points:
1. Your data is private/We respect your privacy.
2. Everything is completely free.
3. a CTA: "Go get it now!"

You would need to remove the huge "PROKEYS" logo from the existing image to make this possible.

----
### **Notes:**

I understand that we haven't yet publicized our mathomania, variables, auto-inserts etc. in these images. The thing is we're limited to four images+one video. Every single person is hankering after snippets. And that's why I think publicizing snippet and everything related to them will give us a better user growth.

---

@iWrote nothing is written in stone and you're free as always to start discussing right away. Also, you need to show at least one image in each promo image (they're supposed to be text+image and not "text only"). So, first tell your thoughts/plans here and only _after_  that start your work.

**Comment by iWrote** [comment]
4 images + 1 video. Alright.

**The Video** is fine. No changes needed. And it's default slide is the CTA. 

**Image One**
I'm thinking two cropped before-after gmail windows with colour-coded highlighting of text and feature explanations where necessary.

1. Maybe I'm not getting this, but why does supporting Rich Text NOT automatically imply Plain Text will also work? 
We could just show some bold/italic text. That'll cover this point without needing to state it.

2. k 

3.  hard to convey what it means, i'll try to include a visual if i can

4.  k 

5. k

3 and 4 need some space to show.  *THEY* didn't really show their context menu. 
(I'm assuming we're not naming you-know-who because Git is public.)

**Image Two.**
 seems straightforward, okay

**Image Three** 
Text with the Snippets page in the backdrop should be enough.
Including the privacy notice here too might be a good idea if we're showing the snippet storage in the background.

**Image Four**
Will repair, reuse, recycle the CTA slide from video. 
 CTA + FREE FREE FREE + privacy notice repeated

**Comment by GaurangTandon** [comment]
**The video** yeah sure agred.

**Image One**
1. Rich support does NOT imply plain text support as well. Think gmail. it has rich text support, but does it support plain text? I've added plain text to basically mean that users can _edit_ their html content using tags like `b` along with the rich editor.
3. i'd made a typo. it was context menu integration. is the visual really hard?

Yes, _they_ didn't show their context menu. But, _I_ wanted them to be shown if we could. If context menus _cannot_ be shown, just skip them. Don't over-congest the promo image. That looks clumsy.

**Image Two** ok
**Image Three** 
This one has the theme "extreme organization" I don't see how the privacy notice fits here. 
Also, the snippets page is only having _symbols_ for sorting, folders, etc. You'll need to add _text_ in the promo image to let the user know what they mean.

**Image Four**
you missed the "works on all websites + logos of favorite sites" point.

------

PS: you forgot to suggest the theme headline ("extreme --something--") for the 1st and 4th image?

**Comment by iWrote** [comment]
Img1
1. ok
2. i'm not sure, visual might look congested, mentioning it isn't a problem

Img3
Okay. Privacy notice goes to 4 then.

Img4 
Sorry. Missed that line. Will reuse the appImg then.

**Comment by iWrote** [comment]
can i consider 4th done? mailed it to you

**Comment by GaurangTandon** [comment]
keep checking your email :)

**Comment by GaurangTandon** [comment]
Added small detail in promo image 1 details. You'll see it in action in next update. Can that detail fit in promo image? (I think not but I _must_ ask)

**Comment by iWrote** [comment]
PROMO IMAGE 1

![wipcurr](https://user-images.githubusercontent.com/18333895/40731443-b169552a-644e-11e8-8957-9003a2ede2b3.png)
I'm not sure but I think this image was supposed to replace one of the other promo images. I remember making some changes as you'd requested and then forgetting to let you know.

What do we do with this now?

I suggest we replace one of these - 
[Link](https://lh3.googleusercontent.com/NXQqH5IrcezlUZNpAsq9QERBgY26E2OA4MBqDdnoAr3VUfA6wYchHVJw4c7zRrFOA_kuftqRYQ=w640-h400-e365)
[Link](https://lh3.googleusercontent.com/HYnVnD3HYJYqKY6uhtAgVVByJtu7GA9NVlBEbVbgFtnYDGq2fSlBnapPHEeCLoPv2ZFP-kx-2A=w640-h400-e365)
with the new one.

Or are there any changes you suggest? Or had we decided not to use this image at all?

**Comment by GaurangTandon** [comment]
![swiffknifeimproved](https://user-images.githubusercontent.com/6308683/41105128-933e00f4-6a8a-11e8-88a2-9267fe7c1adf.png)

Okay, I made these changes:

1. centered the bullet dot on the right unordered list
2. changed "60% faster"<sup>@iWrote [citation needed]</sup> to "context menu integration" (with a different font. I spent 10 minutes on whatfontis.com without success)
3. _almost_ fixed the background for the "Keep calm" image. I notice that you've used a gradient in the original image, so it isn't still proper.

**Comment by iWrote** [comment]
0. I appreciate the efficiency and intended helpfulness but I'm kinda uncomfortable with you quickly wrapping up my unfinished WIP image using a compressed .png version and recompressing it . . . 

0.5 Explaining [0] so I don't come off as hostile :P

### Why you MUST at least let me send you the **.pdn** if you want to edit something -

- I keep everything in layers. So the bg for the meme was _perfectly_ fixed in under 5 seconds. This was just one of the things I never got round to before we went on hiatus and couldn't notice unzoomed when I came back to it after ages.
- I keep an invisible notes layer that mentions all fonts and sizes so I can keep things consistent. So you didn't need to search for it.
- The textboxes are in layers and sublayers to be easily editable. (You reintroduced a colour-block-against-gradient when you changed that text on the arrow, same as the keep-calm-bg-issue)
-  Bullet points aren't vertically centred when text wraps to a new line. Markdown here follows the same standard. The bullet for this text spanning two lines is in the line above.
- Editing from a compressed .jpg and recompressing it can do weird things like changing colours, blur, blah
- Using the wand tool to manually unmerge layers gives undesireable artefacts like jagged edges, thinned text (see point 2 and 4 of the featurelist on your revision), etc.

**TL;DR  plz lemme dew it or ask for .pdn 'cuz LAYERZ are kewl and efficient**

1. addressed 
2. rephrased to explain ctx menu integ and avoid redundancy
3. done
4. Attached .pdn if you wish to make changes, save as .png with "Auto/32-bit" quality setting

![asdf](https://user-images.githubusercontent.com/18333895/41195239-f335cee8-6c46-11e8-9c3b-51ea5e432e09.png)



[TEH PROPER FILE.zip](https://github.com/GaurangTandon/ProKeys/files/2087263/TEH.PROPER.FILE.zip)

**Comment by JAIDEV123** [comment]
Love the ideas here. Will look into it deeper but I still feel that the ProKeys description and images are going to overwhelm users. Maybe I've taken the whole "Users are are stupid as u can imagine" principle to the extreme. Example: showing a rich text placeholder with dates, and just one heading promoting placeholder text.

**Comment by GaurangTandon** [comment]
@JAIDEV123 alright, I'm fine with that direction too. Just note that we only have 5 slots for promo tiles on the webstore, one of which is taken by the video. So, with just four slots, we should be able to communicate the most important points.
